portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +28,56 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private List<String> quotes;
+    private List<String> jstrings;
+
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Si se Puede");
+    quotes.add("Shout out to my mom");
+    quotes.add("Siempre Humble");
+    quotes.add("Adelante y con ganas");
+    quotes.add("Don't complain you have too much on your plate when your goal was to eat");
+    quotes.add("She believes she could so she did");
+
+    jstrings = new ArrayList<>();
+    jstrings.add("I hope this string wroks");
+    jstrings.add("And also this string");
+    jstrings.add("This the third string");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String quote = quotes.get((int) (Math.random() * quotes.size()));
+        
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Ricardo!</h1>");
+    response.getWriter().println("Hello Ricardo!");
+    response.getWriter().println(quote);
+
+    
+    DataServlet dataServlet = new DataServlet(); // What parameter do I pass here, is it the jstrings array?
```

Comment:
A couple suggestions:

1. Use convertToJsonUsingGson instead, since that method calls the gson library which handles converting Java objects into JSON strings.
2. Instead of passing DataServlet to convertToJson, pass in either quotes or jstrings. The gson library will convert your array into a JSON string representation of an array. Make sure to modify script.js to handle this new array.